Makefile

```diff
@@ -5,7 +5,7 @@ endif
 .SECONDARY:
 .PHONY: all clean
 all: apycula/GW1N-1.pickle apycula/GW1N-9.pickle apycula/GW1N-4.pickle \
-	 apycula/GW1NS-2.pickle apycula/GW1NS-4.pickle
+	 apycula/GW1NS-2.pickle apycula/GW1NS-4.pickle apycula/GW1N-9C.pickle
```

Comment:
A separate base from the GW1N-9? If the part number doesn't carry something to distinguish between these chips, then we'll have to change the nextpnr cli first, since it seems to me that it will pick up the GW1N-9 base now, right?

Reply:
Yes, the cli parameter should be changed. 🥲

From the `device_info.csv`, it has two `GW1NR-LV9QN88PC6/I5`

```csv
gw1nr9-012,GW1NR-LV9QN88PC6/I5,GW1NR,GW1NR-9,QFN88P,LV,C6/I5,6,71,8640,6480,17280bit,468Kb,608Kb,20,2,4
gw1nr9c-004,GW1NR-LV9QN88PC6/I5,GW1NR,GW1NR-9C,QFN88P,LV,C6/I5,6,71,8640,6480,17280bit,468Kb,608Kb,20,2,4
```

In `gw_sh` tool, `set_device` command is used with a `-name` argument to precisely specify the device.

```tcl
set_device GW1NR-LV9QN88PC6/I5 -name GW1NR-9C
```

Reply:
Actually, they are using different `IDCODE`.

Reply:
The device is distinguished by the third line of silk printing, with a trailing`C`

    [2-digit-YEAR][2-digit-WEEK]C

Reply:
Yea the C devices tend to have their own chipdb https://github.com/YosysHQ/apicula/blob/master/doc/device_grouping.md